### ISSUE TITLE
[QUEST] Into Occupied Territory, [Dungeon] Book of Ur, Items Take Forever to Respawn

### ISSUE THREAD

**Issue by ModestEgoist** [dalaranwow/dalaran-wow#4835]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I was questing with my friend, and one of the things I've noticed is the amount of time it takes certain items or NPCs to respawn if doing a collection quest. For example, the quest Into Occupied Territory (http://www.wowhead.com/quest=9163) where you need to gather the three Night Elf Plans. Each time one party member retrieved the item, it would take along five minutes for it to respawn. I had to abandon it so we could maintain a good playing pace, and he RAF bonus made it negligible. I still want to bring it up because it also affects Dungeon quests, such as the Book of Ur quest (http://www.wowhead.com/object=36738/the-book-of-ur). One of our party members looted the book, and we had to continue on to kill the final boss and then run back just so one more person can lot the book again.

**Expected behaviour**: Tell us what should happen instead
Objects that are looted for quests should be on a quicker respawn timer just so that quests can be completed with a group without having us camp out for five minutes per person.

**Steps to reproduce the problem**:

1. Pick up quest Into Occupied Territory, it's the first quest where I noticed this.
2. Have one person in group loot the items and then time how long it takes for the item to respawn.
3. Try the same with the Book of Ur quest, or any other quest that requires you to loot items that need to respawn.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Second issue with book of ur was fixed in https://github.com/dalaranwow/dalaran-wow/issues/2230
First issue with the plans like https://wowgaming.altervista.org/aowow/?object=181138 is intended and blizzlike.

source:

http://de.wowhead.com/object=181139/nachtelfenplane-anowyn#english-comments
http://de.wowhead.com/object=181139/nachtelfenplane-anowyn#comments:id=398791

Von greystoker (4,414 –  13·43)  am 2008.10.07 (Patch 3.0.1) |   |  
I dont know about anyone else, but I'm sittin on a 20 minute respawn for this one so far. The other 2 have spawned twice (i took orig, son took 2nd, they're there again)
